core/backend/src/ExportGraphics.ts

```diff
@@ -204,6 +218,20 @@ export interface ExportPartGraphicsOptions {
   angleTol?: number;
   /** Max length of any edge in generated faces, see [StrokeOptions]($geometry-core) */
   maxEdgeLength?: number;
+  /** The longest dimension of a line style's largest component must be at least this size in order for

```

Comment:
Do the two new properties take the scale of the part instance into account?

Reply:
Yes, that's incorporated under the hood so both values can be specified in meters.